.github/workflows/docker-build-push.yml

```diff
@@ -1,17 +1,27 @@
 name: docker-build-push
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - gce-deploy # for testing purposes
+
+env:
+  PROJECT_ID: ${{ secrets.GCE_PROJECT }}
+  GCE_INSTANCE: deploy-pudl-vm
+  GCE_INSTANCE_ZONE: us-central1-a
 
 jobs:
   push_to_registry:
     name: Build Docker image and push to Docker Hub
     runs-on: ubuntu-latest
+    outputs:
+      docker_tags: ${{ steps.docker_metadata.outputs.tags }}
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3
 
       - name: Docker Metadata
         id: docker_metadata
-        uses: docker/metadata-action@v3.8.0
+        uses: docker/metadata-action@v4.0.1
```

Comment:
I saw that these versions got updated earlier today by the dependabot on `dev` so I rejected the automated PRs and added the new action versions here.